mlflow/ml_package_versions.py

```diff
@@ -266,6 +266,15 @@
             "maximum": "0.1.9"
         }
     },
+    "promptflow": {
```

Comment:
@brynn-code This file is not intended to updated manually.

> This file was auto-generated by update_ml_package_versions.py.
> Please do not edit it manually.

Instead, we have to update [ml-package-versions.yml](https://github.com/mlflow/mlflow/blob/master/mlflow/ml-package-versions.yml). Updating the file also enables CI tests for the promptflow flavor. Could you remove the diff in this file and instead up date the YAML file accordingly?

A few additional fields need to be added,
1. `package_info/install_dev` field specifies a command that installs the latest 'dev' version of the library. This is used by our CI tests that proactively runs test against the latest commit so we can detect compatible issue earlier. Typically this is pip install from the git master branch.
2. `models/run` field specifies the pytest command to run the tests for the flavor.
3. `models/requirements` field lists required packages to run the test.

Reply:
Thank you for the heads up, sorry that I didn't notice about the warning message, I'll update it.

Reply:
Np we didn't notice that as well:) Thanks for the fix!

Reply:
Updated and test run successfully.